libraries/botbuilder-dialogs-adaptive/src/recognizers/adaptiveRecognizer.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * @module botbuilder-dialogs-adaptive
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { BoolExpression } from "adaptive-expressions";
+import { getTopScoringIntent, RecognizerResult } from "botbuilder-core";
+import { DialogContext, Recognizer } from "botbuilder-dialogs";
+
+// TODO
+// (differs from C# in that it uses <name-of-recognizer>Configuration interfaces)
+// AdaptiveRecognizerConfiguration interface
+// with optional BoolExpression logPersonalInformation
+
+export abstract class AdaptiveRecognizer extends Recognizer {
+    /**
+     * (Optional) Flag that designates whether personally identifiable information (PII) should log to telemetry.
+     */
+    public logPersonalInformation = new BoolExpression('=settings.telemetry.logPersonalInformation');
+
+    /**
+     * Uses the RecognizerResult to create a list of propeties to be included when tracking the result in telemetry.
+     * @param recognizerResult Recognizer Result.
```

Comment:
Our JSDoc eslint complains when types are omitted - can you add those? There should also be a newline between the description and the parameters.

Reply:
Done. Added BoolExpression type